.github/workflows/reset-versions.yml

```diff
@@ -0,0 +1,43 @@
+name: Reset versions in release-please-config.json
+on:
+  release:
+    types: published
```

Comment:
we publish a lot of releases per release PR - I think you'll want to filter on one something [like this](https://github.com/looker-open-source/sdk-codegen/blob/3ae42cd97f974322b77d58941ac72b88be3f8b97/.github/workflows/lerna-publish.yml#L17)